FindSplash.cmake

```diff
@@ -96,7 +102,7 @@ if(Splash_ROOT_DIR)
     #
     find_library(Splash_LIBRARIES
       NAMES splash
-      PATHS $ENV{SPLASH_ROOT}/lib)
+      PATHS ${Splash_ROOT_DIR}/lib)
```

Comment:
It is not a hint is is the variable holding the actual root path. And I did not come up with the name. Additionally I wouldn't remove the hint (`SPLASH_ROOT`) for now/in this PR